src/components/rows/FiveStoriesOneAdBundle/index.js

```diff
@@ -1,126 +1,39 @@
 import React from "react";
 import PropTypes from "prop-types";
-import get from "lodash/get";
-import CollectionLink from "../../utils/generate-collection-link";
 import { StoryCardML4x3DP1x1 } from "../../atoms/StoryCardML4x3DP1x1";
-import { StoryCardML4x3DP7x5HeadlineOverlay } from "../../atoms/StoryCardML4x3DP7x5HeadlineOverlay";
 import { StoryCardML8x5DP7x5HeadlineOverlay } from "../../atoms/StoryCardML8x5DP7x5HeadlineOverlay";
 
-import ResponsiveImageWithFallback from "../../atoms/ResponsiveImageWithFallback";
-import Headline from "../../atoms/Headline";
-
-import styles from "./fiveStoriesOneAdBundle.m.css";
-import { generateImageSources, getStoryHeadline } from "../../utils/utils";
+import "./fiveStoriesOneAdBundle.m.css";
 
 export const FiveStoriesOneAdBundle = ({ collection, accentColor, pagetype = "" }) => {
-  const numberOfStories = get(collection, ["total-count"], 0);
-  const stories = collection.items
-    .filter(({ story = {} }) => getStoryHeadline(story) && story.url)
-    .slice(0, 5)
-    .map(({ story }) => story);
+  // const numberOfStories = get(collection, ["total-count"], 0);
+  const stories = collection.items.slice(0, 8).map(({ story }) => story);
   return (
     <div styleName="base" style={accentColor && { "--accent-color": accentColor }}>
-      <div styleName="row" className="container row">
-        <div styleName="bundle-header" className="row">
-          <CollectionLink collection={collection} className={styles["bundle-header"]}>
-            <Headline
-              text={collection.name}
-              headerType={10}
-              headerLevel={2}
-              headlineDesign={"crossline"}
-              className={"collection-title"}
-            />
-          </CollectionLink>
-          <div styleName="bundle-summary-wrap">
-            {numberOfStories && <span styleName="total-count"> {numberOfStories + " அளவுக்கு"}</span>}
-            <p styleName="bundle-summary">{collection.summary}</p>
-          </div>
-        </div>
+      <div styleName="row" className="row">
         <div styleName="col-3">
-          <CollectionMeta collection={collection} />
+          {stories.splice(0, 2).map(story => (
```

Comment:
Component is `FiveStoriesOneAdBundle` but has 7 stories one ad and is marked under 5 stories in the story book.

Reply:
have grouped under 7 stories now